### PR TITLE
release yaml for v1.21.0

### PR DESCRIPTION
**Pull request by areshand**
## Description

cargo run --profile performance -p aptos-release-builder -- generate-proposals --release-config aptos-move/aptos-release-builder/data/release.yaml --output-dir output --simulate testnet

```
RemoteFile(Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("raw.githubusercontent.com")), port: None, path: "/aptos-labs/aptos-networks/main/gas/v1.21.1.json", query: None, fragment: None })
RemoteFile(Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("raw.githubusercontent.com")), port: None, path: "/aptos-labs/aptos-networks/main/gas/v1.20.0.json", query: None, fragment: None })
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING execution_hash
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING execution_hash
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING execution_hash
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING execution_hash
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING execution_hash
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING execution_hash
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING execution_hash
Found 1 proposal
    output/sources/v1.21.1/step_1_upgrade_framework
        0-gas-schedule.move
        1-move-stdlib.move
        2-aptos-stdlib.move
        3-aptos-framework.move
        4-aptos-token.move
        5-aptos-token-objects.move
Simulating proposal at output/sources/v1.21.1/step_1_upgrade_framework
Compiling scripts...
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING script
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING script
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING script
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING script
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING script
Compiling, may take a little while to download git dependencies...
INCLUDING DEPENDENCY AptosFramework
INCLUDING DEPENDENCY AptosStdlib
INCLUDING DEPENDENCY MoveStdlib
BUILDING script
Creating and funding sender account.. done
Executing governance scripts...
    0-gas-schedule.move
        Success
    1-move-stdlib.move
        Success
    2-aptos-stdlib.move
        Success
    3-aptos-framework.move
        Success
    4-aptos-token.move
        Success
    5-aptos-token-objects.move
        Success
All scripts succeeded!
All proposals succeeded!
```

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Move Compiler
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by sherry-x — APPROVED** [review]

**Comment by areshand** [comment]
@vgao1996  @zekun000  @igor-aptos  could you pls approve this PR ?

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -1,19 +1,17 @@
 ---
-remote_endpoint: https://fullnode.mainnet.aptoslabs.com
-name: "TBD"
+remote_endpoint: ~
+name: "v1.21.0"
 proposals:
-  - name: proposal_1_upgrade_framework
+  - name: step_1_upgrade_framework
```
Comment (vgao1996) [inline aptos-move/aptos-release-builder/data/release.yaml:5]: `proposal_` is a more accurate name here.

Technically `steps` refer to the items under `update_sequence`

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
-      title: "Multi-step proposal to upgrade mainnet framework, version TBD"
-      description: "This includes changes in (TBA: URL to changes)"
+      title: "Multi-step proposal to upgrade mainnet framework, version v1.21.0"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.21.1-rc and https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.21.0-rc"
     execution_mode: MultiStep
     update_sequence:
       - Gas:
-          new: current
+          old: https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/gas/v1.20.0.json
+          new: https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/gas/v1.21.1.json
```
Comment (igor-aptos) [inline aptos-move/aptos-release-builder/data/release.yaml:13]: I think you need to change this to 1.21.0 too :)

Comment (areshand) [reply aptos-move/aptos-release-builder/data/release.yaml:13]: This is generated from the latest v1.21.1 binary. Does the name of gas schedule version named after the binary version or node version doesn't matter? cc: @sherry-x

**Review by igor-aptos — APPROVED** [review]

**Pull request merged**